integration/cri-containerd/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# 
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -eE
+
+err_report() {
+	cat /tmp/test-cri/cri-containerd.log
+	cat /tmp/test-cri/containerd.log
+}
+
+trap 'err_report' ERR
+
+
+repo="github.com/containerd/cri-containerd"
+# FIXME: cri-containerd fails using macvtap.
+sudo sed -r -i -e 's/^(internetworking_model=).*/\1"bridged"/g' /usr/share/defaults/clear-containers/configuration.toml
```

Comment:
Might be worth creating a variable for this path (atleast the `clear-containers/` element) to make it easier to migrate it to Kata?